cebra/data/single_session.py

```diff
@@ -261,27 +261,47 @@ class MixedDataLoader(cebra_data.Loader):
     1. Positive pairs always share their discrete variable.
     2. Positive pairs are drawn only based on their conditional,
        not discrete variable.
+
+    Args:
+        conditional (str): The conditional variable for sampling positive pairs. :py:attr:`cebra.CEBRA.conditional`
+        time_offset (int): :py:attr:`cebra.CEBRA.time_offsets`
+        positive_sampling (str): either "discrete_variable" (default) or "conditional"
+        discrete_sampling_prior (str): either "empirical" (default) or "uniform"
     """
 
     conditional: str = dataclasses.field(default="time_delta")
     time_offset: int = dataclasses.field(default=10)
+    positive_sampling: str = dataclasses.field(default="discrete_variable")
+    discrete_sampling_prior: str = dataclasses.field(default="uniform")
```

Comment:
these new options should be covered by a test.

Reply:
I added a test that covers the different parameters here [1db5152](https://github.com/AdaptiveMotorControlLab/CEBRA/pull/101/commits/1db5152ccbcc3707848a7b101e2b88593f098e56). I can also extend it, for example testing if e.g. a wrong parameter got passed, and pytest.raises catches ValueError if you think that's useful.